js/sudoku.js

```
@@ -0,0 +1,243 @@
+					}
+					//add rows to table
+					defaults.table.append(tempRow);
+				}
+				//add extra div in here for background decoration
+				defaults.table.append("<div class='sdk-table-bk'></div>");
+				//add table to screen
+				$this.append(defaults.table);
+				
+				var items = defaults.level;
+				while (items > 0) {
+					var row = Math.floor(Math.random() * 9);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'row'. Consider to delete it or rename it.

'row' is defined here:


(lint/suspicious/noRedeclare)